src/chum.cpp

```diff
@@ -29,9 +28,16 @@ static inline auto role2operation(Transaction::Role role) {
 }
 
 Chum::Chum(QObject *parent)
-  : QObject{parent} {
+  : QObject{parent}
+{
+  connect(&m_ssu, &Ssu::updated, this, &Chum::repositoriesListUpdated);
+  connect(&m_ssu, &Ssu::updated, this, &Chum::repoUpdated);
   connect(Daemon::global(), &Daemon::updatesChanged, [this]() { this->getUpdates(); });
-  refreshPackages();
+
+  m_busy = true;
+  //% "Load repositories"
+  SET_STATUS(qtTrId("chum-load-repositories"));
```

Comment:
Wondering why we bother having this as a macro ..... compiler probably optimises away if necessary anyway

Reply:
Its easier to write 1 line instead of 3. As we need to emit the signal, it avoids it when status stayed the same

Reply:
I mean, set_status could just be a regular function, and the code would remain as-is

Reply:
I missed this comment, sorry! Yes, it makes sense to have it as a function - I will make the change and let's see where it will make it.